components/authz-service/storage/v2/postgres/postgres_test.go

```diff
@@ -6438,9 +6567,10 @@ func insertAppliedRule(t *testing.T, db *testhelpers.TestDB, rule storage.Rule)
 			dbID, pq.Array(c.Value), c.Attribute.String(), c.Operator.String())
 		require.NoError(t, err)
 	}
+	rule.Status = "applied"
 }
 
-func insertStagedRule(t *testing.T, db *testhelpers.TestDB, rule storage.Rule) {
+func insertStagedRule(t *testing.T, db *testhelpers.TestDB, rule *storage.Rule) {
```

Comment:
ℹ️ We have to change the function type to pass a pointer to mutate the `rule` argument, to add the status below